src/bussinessLogic/ManageArray.java

```diff
@@ -42,35 +39,12 @@ public ManageArray(MapManager manageMap) {
         this.manageMap = manageMap;
     }
 
-    /**
-     * prepare all the data necessary for the evaluation phase
-     */
-    public void manageEvaluationData() {
-        setCharArray(alphabetMultiplier);
-        shuffleCharArray(evaluationArray);
-        manageMap.setNewCharMap();
-    }
-
-    /**
-     * prepare all the data necessary for the practice phase
-     */
-    public void managePracticeData() {
-        calculateAverage();
-        manageMap.sortCharMap();
-        setCharArrayFromSortedMap();
-        // reset the charMap to store new data
-        manageMap.setNewCharMap();
-        setPracticeArray();
-        shuffleCharArray(practiceArray);
-    }
-
     /**
      * generate an array containing all letters for a number of alphabetMultiplier times
      *
      * @param alphabetMultiplier
      */
-    private void setCharArray(int alphabetMultiplier) {
-        this.alphabetMultiplier = alphabetMultiplier;
+    public void setEvaluationArray(int alphabetMultiplier) {
         evaluationArray = new char[numberOfLetters * alphabetMultiplier];
```

Comment:
This initialization can be done when declaring this property at class level.

Reply:
👍